src/main/java/taxi/controller/car/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,31 @@
+package taxi.controller.car;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.service.CarService;
+
+@WebServlet(urlPatterns = "/drivers/my")
```

Comment:
it would be better to use 
```suggestion
@WebServlet(urlPatterns = "/drivers/cars")
```